Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,702 @@
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      searchText:
+        complex:
+          root: inputs.Hostname
+          transformers:
+          - operator: uniq
+      verbose: {}
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -650,
+          "y": 996
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "8":
+    id: "8"
+    taskid: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+    type: condition
+    task:
+      id: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+      version: -1
+      name: Is Carbon Black Enterprise Response enabled?
+      description: Checks if there is any active instance of the Carbon Black Enterprise
+        Response integration enabled.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "9"
+    scriptarguments:
+      value:
+        complex:
+          root: modules
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: brand
+                iscontext: true
+              right:
+                value:
+                  simple: carbonblack-v2
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: state
+                iscontext: true
+              right:
+                value:
+                  simple: active
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -180,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "9":
+    id: "9"
+    taskid: 79f17c3a-2456-47d9-8e68-cc3d84ef01bd
+    type: regular
+    task:
+      id: 79f17c3a-2456-47d9-8e68-cc3d84ef01bd
+      version: -1
+      name: Get host information from Carbon Black Enterprise Response
+      description: Gets host information from Carbon Black Enterprise Response.
+      script: carbonblack-v2|||cb-sensor-info
+      type: regular
+      iscommand: true
+      brand: carbonblack-v2
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      groupid: {}
+      hostname:
+        complex:
+          root: inputs.Hostname
+          transformers:
+          - operator: uniq
+      ip: {}
+      sensor: {}
+    reputationcalc: 1
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -180,
+          "y": 996
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "10":
+    id: "10"
+    taskid: 216aed2b-2f34-4c5e-8c54-60b978218896
+    type: condition
+    task:
+      id: 216aed2b-2f34-4c5e-8c54-60b978218896
+      version: -1
+      name: Is CrowdStrike Falcon Host enabled?
+      description: Checks if there is any active instance of the CrowdStrike Falcon
```

Comment:
Checks if there is an active instance of the CrowdStrike Falcon Host integration enabled.